README.md

```diff
@@ -1,11 +1,72 @@
 [![Build Status](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client.svg?branch=master)](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client)
 
-pint
-====
+# Introduction
 
 pint (Public Cloud INformation Tracker) is a command line client to access
 the REST API provided by the SUSE Public Cloud Information Service. The
 information service provides data about the images SUSE publishes in various
 public cloud frameworks (Amazon, Google, Microsoft) as well as the
 servers that make up the SUSE maintained and operated update infrastructure
 in those frameworks.
+
+## Quick Start
+
+There are various ways to install the Pint Client:
+
+### Install the python package using pip
```

Comment:
I would recommend creating a virtrualenv in which to do the pip installation; installing a Python module via pip directly into the system Python infrastructure can corrupt that infrastructure, and in the case of SLE, could invalidate the customer's support contract. ;-)

Reply:
agreed

Reply:
Potentially also suggest that this is the recommended install mechanism for the tool on non-SUSE distros, e.g. Debian/Ubuntu or RHEL/CentOS.